activemodel/lib/active_model/dirty.rb

```diff
@@ -54,7 +58,7 @@ module ActiveModel
   #
   # A newly instantiated object is unchanged:
   #
-  #   person = Person.find_by(name: 'Uncle Bob')
+  #   person = Person.new 'Uncle Bob'
```

Comment:
I was tied up between the two hence had opened this up for discussion. I liked the first one, since a standalone example is better.